tutorial/cache_pyh_files.md

```diff
@@ -0,0 +1,33 @@
+# Picklize header files
+
+## header files
+
+A header file in pyccel is a file containing function or variable declarations, macro definitions , templates and metavariable declarations.
+the convention is to give header files names that end with `.pyh` .
```

Comment:
Is this a convention or a rule? A convention is something that is usually done but doesn't have to be whereas a rule needs to be followed

Reply:
it's a rule okay I will change it